ui/app/css/itcss/components/send.scss

```diff
@@ -70,4 +79,117 @@
   justify-content: space-between;
 }
 
-.send-screen-bolt-icon {}
\ No newline at end of file
+.send-screen-bolt-icon {}
```

Comment:
remove

Reply:
removed in https://github.com/danjm/metamask-extension/tree/NewUI-flat-lint-fixes, which I will make a new PR for after this one